### PR TITLE
Nuke log

### PR DESCRIPTION
**Pull request by AusarAtkin**
## About The Pull Request

Good log, bad unlogged critical events.
## Why It's Good For The Game

Improve griffer visibility
## Changelog
:cl:
code: improve nuke logging
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/nuclearbomb.dm [hunk 1]
```diff
@@ -65,12 +65,16 @@
 	update_minimap_icon()
 	// The timer is needed for when the signal is sent
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_NUKE_START, src)
+	log_game("[usr] has enabled the nuke at [AREACOORD(src)]")
```
Comment (TiviPlus) [inline code/game/objects/machinery/nuclearbomb.dm:68]: do not use usr, it is a very innacurate byond var that should only be used in a few niche cases. pass the user to the proc as an argument properly instead

File: code/game/objects/machinery/nuclearbomb.dm [hunk 2]
```diff
@@ -65,12 +65,16 @@
...
 
 ///Disables nuke timer
 /obj/machinery/nuclearbomb/proc/disable()
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_NUKE_STOP, src)
 	countdown.stop()
 	GLOB.active_nuke_list -= src
+	if(timer_enabled)
+		log_game("[usr] has disabled the nuke at [AREACOORD(src)]")
+		message_admins("[usr] has disabled the nuke at [ADMIN_VERBOSEJMP(src)]") //Incase disputes show up about marines griefing and the like.
```
Comment (TiviPlus) [inline code/game/objects/machinery/nuclearbomb.dm:77]: ditto

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/nuclearbomb.dm (hunk 1 above)
Comment (AusarAtkin) [reply code/game/objects/machinery/nuclearbomb.dm]: doen

File: code/game/objects/machinery/nuclearbomb.dm (hunk 2 above)
Comment (AusarAtkin) [reply code/game/objects/machinery/nuclearbomb.dm]: donetto

**Comment by despong** [comment]
Absolute Cinema Bluesummers Throw

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**